client/src/contentful-common-strings/pull.js

```diff
@@ -0,0 +1,71 @@
+//@ts-check
+
+/**
```

Comment:
Did you want to leave this comment in here? Is this some kind of Javascript typing convention I'm not aware of?

Reply:
Oh yeah, so `//@ts-check` is a magical comment that lets TypeScript know that even that this is a JS file, it (and its jsdoc comments) should be type-checked.  (I didn't write this in TS because I didn't want to have to deal with running it through a transpile pipeline to run it from the command-line, but I also wanted some of the benefits of TS, which TS happily provides--see their docs on [JS projects utilizing TypeScript](https://www.typescriptlang.org/docs/handbook/intro-to-js-ts.html) for more info!)